From 5a52cae54a05499a8487f392cf5dfc3d8a837e6f Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 20 Oct 2023 17:12:09 +0100
Subject: [PATCH] dt: Switch bcm2712 firmware-kms node to using the 2712
 compatible

With the new compatible to handle the interrupts correctly, switch
the base dt to use it.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2712.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/bcm2712.dtsi
+++ b/arch/arm/boot/dts/bcm2712.dtsi
@@ -103,7 +103,7 @@
 		};
 
 		firmwarekms: firmwarekms@7d503000 {
-			compatible = "raspberrypi,rpi-firmware-kms";
+			compatible = "raspberrypi,rpi-firmware-kms-2712";
 			/* SUN_L2 interrupt reg */
 			reg = <0x7d503000 0x18>;
 			interrupt-parent = <&cpu_l2_irq>;
